src/utils/mediaQueries.ts

```diff
@@ -1,6 +1,8 @@
 import { Command } from 'types'
 
-export const MEDIA_QUERY_MATCHES = [
+export type Breakpoints = 'xl' | 'lg' | 'md' | 'sm' | 'xs'
+
+export const MEDIA_QUERY_MATCHES: Array<{ name: Breakpoints; query: string }> = [
```

Comment:
@ramirotw, @alongoni I have found it difficult to get the desired behaviour when following the screen changes, as there are these media queries and there are others that are used with the theme that are much more detailed in [`theme/styles/media.ts`](https://github.com/cowprotocol/explorer/blob/74-surplus-on-tx-table/src/theme/styles/media.ts#L1). 

Do you think we can unify this?

Reply:
how do you suggest unifying both files? Agreed that it doesn't make sense to have two separate breakpoints definitions

Reply:
The idea would be to use `src/utils/mediaQueries.ts` as the main source, but because changing this file may affect whoever uses MEDIA_QUERY_MATCHES It will be better to handle it in #102